modules/currency-l1/src/test/scala/org/tessellation/currency/l1/http/DataApplicationRoutesSuite.scala

```diff
@@ -177,6 +194,31 @@ object DataApplicationRoutesSuite extends HttpSuite {
     }
   }
 
+  test("POST /data returns OK . . .") {
```

Comment:
what does `. . .` mean?

Reply:
@marcinwadon How about this?
`"POST /data returns OK and response as JSON object for valid updates"`

Reply:
do you check whether it's a valid JSON object or you check for the specific schema?
maybe: `POST /data returns 200 Ok and update's hash as a json object`